### ISSUE TITLE
Prometheus Rule For Missed Block Alerts

### ISSUE THREAD

**Issue by HamidullahMuslih** [kilnfi/eth-validator-watcher#102]
I am using the below expression to alert when my validator misses a block.
```
increase(missed_block_proposals_head_count_total{}[12h]) > 1
``` 
but seems there are too many false positive alerts.

My questions are:
1- Does the above metric count missed blocks just for my validators or for all the  network validator?
2- Which metric expression I should use in Prometheus to get alerted when my validator misses a block? 

Thanks!

**Comment by aimxhaisse** [comment]
Hello,

`missed_block_proposals_head_count_total` uses the head of the network as seen by your beacon, it is not finalized and as a result, blocks you see as missed can actually be re-orged and make it, you can use `missed_block_proposals_finalized_count_details_total` to use the finalized view of the network, this lags a bit behind (~2 epochs during normal conditions) but shouldn't yield false positives.

We use it as follows at Kiln:

```        round(increase(missed_block_proposals_finalized_count_total[2h])) > 1
```

**Comment by HamidullahMuslih** [comment]
Thanks @aimxhaisse for the prompt response, I will try this 🙏

**Comment by hamidmuslih** [comment]
Hello @aimxhaisse, I have applied your suggested rule, and today there was one orphaned missed block, unfortunately, it didn't alert us. does this rule also pick the orphaned missed blocks? 

Below is my stack. 
Kiln version:  `ghcr.io/kilnfi/eth-validator-watcher:0.7.1`
Geth version: `1.13.14`
Lighthouse version: `v5.1.3`

Below are the flags passed to Kiln 
```
                "--beacon-url",
                "http://xxxxxxxx:4000",
                "--beacon-type",
                "lighthouse",
                "--execution-url",
                "http://xxxxxxx:8545",
                "--pubkeys-file-path",
                "/app/keys.txt",
                "--relay-url",
            .......
```

The previous rule that I was using could catch orphaned missed blocks, but it had some false positive alerts. 

thanks.

**Comment by aimxhaisse** [comment]
Hello,

Yes in the current form it misses orphan blocks, I'm adding it in the next version being worked on.

**Comment by hamidmuslih** [comment]
yeah one time it caught orphaned block with `.... > 1` rule, I will update it to `... > 0` thanks
